### PR TITLE
Pydra implementation of t1volume-create-dartel

### PR DESCRIPTION
**Pull request by omar-rifai**
Pydra implementation of t1volume-create-dartel. Replaces #739

### CODE REVIEW

**Comment by omar-rifai** [comment]
I'm not sure what the issue with the CI is but I tried running the instantiation tests locally and they run successfully.  
image

File: clinica/utils/inputs.py [hunk 1]
```diff
@@ -267,28 +267,52 @@ def find_sub_ses_pattern_path(
 
 
 def _check_information(information: Dict) -> None:
-    if not isinstance(information, dict):
-        raise TypeError("A dict must be provided for the argument 'information'")
-
-    if not all(elem in information for elem in ["pattern", "description"]):
-        raise ValueError(
-            "'information' must contain the keys 'pattern' and 'description'"
+    if not isinstance(information, (dict, list)):
```
Comment (NicolasGensollen) [inline clinica/utils/inputs.py:270]: Could you update the function's signature to reflect that `information` can be a list or a dict?

File: clinica/utils/inputs.py [hunk 2]
```diff
@@ -267,28 +267,52 @@ def find_sub_ses_pattern_path(
...
-    if not all(
-        elem in ["pattern", "description", "needed_pipeline"]
-        for elem in information.keys()
-    ):
-        raise ValueError(
-            "'information' can only contain the keys 'pattern', 'description' and 'needed_pipeline'"
-        )
+    if isinstance(information, list):
+        for item in information:
+            if not all(elem in item for elem in ["pattern", "description"]):
```
Comment (NicolasGensollen) [inline clinica/utils/inputs.py:277]: Lines 277 to 295 are copy pasted below. Could you turn them into a function (`_check_information_dict`??) and simply call it for all items in the list (`[ _check_information_dict(elt) for elt in information]`)?

**Review by NicolasGensollen — COMMENTED** [review]
Thanks @omar-rifai !
LGTM and I was able to run the pipeline. 🎉 
I just have a few small suggestions before we merge.

**Comment by NicolasGensollen** [comment]
I'll take care of these in a follow-up PR.
Merging...

**Pull request merged**